templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'taxi:driver-detail' pk=driver.id %}" class="btn btn-secondary">Cancel</a>
```

Comment:
Ensure that the context variable `driver` is passed to this template from the view. This is necessary for the `{% url 'taxi:driver-detail' pk=driver.id %}` tag to work correctly.